.github/workflows/build.yml

```
@@ -248,17 +256,19 @@ jobs:
             --mount "type=bind,src=$PWD/signed,dst=/doi-build/signed"
 
             # https://explore.ggcr.dev/?repo=docker/image-signer-verifier
-            docker/image-signer-verifier:0.3.3@sha256:a5351e6495596429bacea85fbf8f41a77ce7237c26c74fd7c3b94c3e6d409c82
+            "$BASHBREW_IMAGE_SIGNER"
 
             sign
 
-            --envelope-style oci-content-descriptor
-
-            --aws_region "$AWS_KMS_REGION"
-            --aws_arn "awskms:///$AWS_KMS_KEY_ARN"
+            # kms key used to sign attestation artifacts
+            --kms="AWS"
+            --kms-region="$AWS_KMS_REGION"
+            --kms-key-ref="$AWS_KMS_KEY_ARN"
 
-            --input oci:///doi-build/unsigned
-            --output oci:///doi-build/signed
+            --input=oci:///doi-build/unsigned
```

```suggestion
            --input=oci:///doi-build/image
```
We can change the name to anything now that there will be no "signed image" and "unsigned image". This is because all of the attestations are pushed to the referrers repo so the image never mutates.